stable/traefik/values.yaml

```diff
@@ -42,6 +42,10 @@ service:
   #   key: value
 gzip:
   enabled: true
+accessLogs:
+  enabled: false
+  filePath: ""  # if empty, stdout is used
```

Comment:
This will work, but elsewhere in the file, attributes with no default value are commented instead of being set to the empty string. It will have the same effect and will promote consistency within this file.